gitrevise/merge.py

```diff
@@ -201,7 +201,7 @@ def merge_blobs(
 
         # At this point, we know that there are merge conflicts to resolve.
         # Prompt to try and trigger manual resolution.
-        print(f"Merge conflict for '{path}'")
+        print(f"Merge conflict for '{path}', commit '{labels[2]}'")
```

Comment:
We can get away with multi-line prompts, so perhaps let's put the path & commit onto separate lines. Perhaps something like this?

```suggestion
        print(f"Merge conflict while rebasing '{labels[2]}'")
        print(f"  Path: '{path}'")
```

Reply:
Good idea, let's split it.

So you don't want to invent the term "rebase conflict"?
Or avoid calling it anything, like git rebase: `Could not apply '{labels[2]}'`?

Reply:
I used git revise, and changed the commit to this; what do you think?

```suggestion
-        print(f"Merge conflict for '{path}', commit '{labels[2]}'")
+        print(f"Conflict in applying '{labels[2]}'")
+        print(f"  Path: '{path}'")
```

Reply:
Works for me :-)
Thanks!